kicker/src/Application.kt

```diff
@@ -0,0 +1,79 @@
+package com.kicker
+
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import kotlinx.coroutines.runBlocking
+import org.litote.kmongo.Id
+import org.litote.kmongo.coroutine.coroutine
+import org.litote.kmongo.newId
+import org.litote.kmongo.reactivestreams.KMongo
+import java.time.LocalDateTime
+
+data class User(val _id: Id<User>, val nickName: String)
```

Comment:
Please do not use underscores in variable names, as it is generally a discouraged practice.

Please consult KCC https://kotlinlang.org/docs/reference/coding-conventions.html for more details.

If you're using underscores in Mongo as well, you need to change them to camelCase as it is generally used for JSON property names.